pkg/manager/member/tikv_upgrader.go

```diff
@@ -209,6 +209,10 @@ func (u *tikvUpgrader) readyToUpgrade(upgradePod *corev1.Pod, tc *v1alpha1.TidbC
 			klog.Errorf("parse annotation:[%s] to time failed.", EvictLeaderBeginTime)
 			return false
 		}
+		if tc.Status.TiKV.StatefulSet.CurrentReplicas < 2 {
+			klog.Infof("TiKV stores are less than 2, skip waiting to evict region leader for Pod %s/%s", upgradePod.Namespace, upgradePod.Name)
+			return true
+		}
```

Comment:
Can we move this block before L206?

Reply:
Do not use `CurrentReplicas` which is the number of Pods created by the StatefulSet controller from the StatefulSet version indicated by currentRevision. Here is an example during upgrade:
```
status:
  collisionCount: 0
  currentReplicas: 1
  currentRevision: t-tikv-5b4cd9d4df
  observedGeneration: 42
  readyReplicas: 2
  replicas: 3
  updateRevision: t-tikv-8444b66dc8
  updatedReplicas: 2
```

Reply:
BTW, should we move this block to the beginning of this function?

Reply:
Refer to `tc.TiKVStsActualReplicas()`

Reply:
update